### ISSUE TITLE
trianing issues

### ISSUE THREAD

**Issue by VAriana** [WangLibo1995/GeoSeg#3]
I have trained the model for 30 epoch ,but the reults is not so good.
<img width="1014" alt="1652446685(1)" src="https://user-images.githubusercontent.com/102912455/168288426-42184b2b-0b71-4372-8921-8d25646aa8c4.png">
should I change the learning rate much lower?
<img width="467" alt="1652446797(1)" src="https://user-images.githubusercontent.com/102912455/168288612-f16bb907-c9a3-4d77-aaa9-94abebea8389.png">

**Comment by WangLibo1995** [comment]
> I have trained the model for 30 epoch ,but the reults is not so good. <img alt="1652446685(1)" width="1014" src="https://user-images.githubusercontent.com/102912455/168288426-42184b2b-0b71-4372-8921-8d25646aa8c4.png"> should I change the learning rate much lower? <img alt="1652446797(1)" width="467" src="https://user-images.githubusercontent.com/102912455/168288612-f16bb907-c9a3-4d77-aaa9-94abebea8389.png">

Please check whether the pretrained weight is loaded successfully. Some problems you should try to solve yourself instead of always asking questions. It bothers me.
